src/common/GenericDialog.tsx

```diff
@@ -49,7 +49,9 @@ export const GenericDialog = ({
           <ModalCloseButton />
           {header && (
             <ModalHeader fontSize="lg" fontWeight="bold">
-              {header}
+              <Text as="h2" fontSize="xl">
```

Comment:
Let's sort the inconsistency here, though as discussed this code is not used.

Reply:
Made consistent with other uses of the `GenericDialog` component where the heading is included in the dialog body -> [`2a00984`](https://github.com/microbit-foundation/python-editor-v3/pull/1002/commits/2a00984455398ad152a33cf41e9970990fbd825d). 

Use of the header prop changes the appearance significantly, so is not used for the existing connect helper dialogs.